src/sentry/static/sentry/app/views/releasesV2/detail/commits/index.tsx

```diff
@@ -1,8 +1,151 @@
 import React from 'react';
+import {Params} from 'react-router/lib/Router';
+import styled from '@emotion/styled';
 
-type Props = {};
+import AsyncComponent from 'app/components/asyncComponent';
+import CommitRow from 'app/components/commitRow';
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {Repository, Commit} from 'app/types';
+import EmptyStateWarning from 'app/components/emptyStateWarning';
+import {PanelHeader, Panel, PanelBody} from 'app/components/panels';
+import DropdownControl, {DropdownItem} from 'app/components/dropdownControl';
+import overflowEllipsisLeft from 'app/styles/overflowEllipsisLeft';
 
-// TODO(releasesV2): finish this component
-const ReleaseCommits = ({}: Props) => <div>ReleaseCommits</div>;
+import {getCommitsByRepository, CommitsByRepository} from '../utils';
+import ReleaseNoCommitData from '../releaseNoCommitData';
+
+const ALL_REPOSITORIES_LABEL = t('All Repositories');
+
+type Props = {
+  params: Params;
+} & AsyncComponent['props'];
+
+type State = {
+  commits: Commit[];
+  repos: Repository[];
+  activeRepo: string;
+} & AsyncComponent['state'];
+
+class ReleaseCommits extends AsyncComponent<Props, State> {
```

Comment:
Let's use `AsynvView` and add a `getTitle()` so that the browser has a relevant title.

Reply:
I was considering it, but we don't do that with tabs on issue group detail, nor releases v1.
Do we want to change the behavior from now on, or should I leave it like that?

Reply:
Hmm, I think we should do this going forward, otherwise it's hard to use the back button like this: 
![image](https://user-images.githubusercontent.com/79684/76979976-d818c680-68f5-11ea-80fd-1048d68a1361.png)

Reply:
OK, I will refactor all four tabs in separate PR soon.